xdmod/entrypoint.sh

```diff
@@ -40,8 +43,8 @@ then
         xdmod-ingestor -v
     fi
 
-    echo "---> Starting sshd on xdmod..."
-    /usr/sbin/sshd -e
+    echo "---> Setup XDMoD SSO..."
+    /srv/xdmod/scripts/xdmod-setup-sso.sh
```

Comment:
Does this need to execute every time the container is started? If not, perhaps we move this to the initialization part of the entrypoint?